Dockerfile

```diff
@@ -14,7 +14,7 @@ FROM runtime AS production
 LABEL maintainer="HMPPS Digital Studio <info@digital.justice.gov.uk>"
 
 ARG BUILD_NUMBER
-ENV BUILD_NUMBER ${BUILD_NUMBER:-1_0_0}
```

Comment:
given that we're actually fixing this, shouldn't the value be in quotes? or does that mess with the substitution?

Reply:
After a quick search, it looks like we do not require quotes: https://docs.docker.com/reference/dockerfile/#environment-replacement

Reply:
great, TIL!